firestore/tests/field_path_test.cc

```diff
@@ -0,0 +1,210 @@
+// Copyright 2018 Google Inc.
```

Comment:
s/Inc./LLC/

Also, we keep the tests in the same directory as the code they test, please move this to `firestore/client`.